Storage/CertStore.swift

```diff
@@ -23,6 +23,13 @@ open class CertStore {
         return keys.contains(key)
     }
 
+    open func hasAddedCertificate(forOrigin origin: String) -> Bool {
+        return keys.first(where: { key in
+            let subs = key.split(separator: "/")
+            return subs.count > 0 ? (subs[0] == origin) : false
```

Comment:
I don't ever see thing getting called. Either way, I've updated the ticket to include more information for what needs to be done.